pkgs/servers/rippled/validator-keys-tool.nix

```diff
@@ -1,27 +1,29 @@
-{ stdenv, lib, fetchgit, cmake, openssl, boost, zlib, rippled }:
+{ stdenv, lib, fetchFromGitHub, cmake, openssl, boost, zlib, icu, rippled }:
 
 stdenv.mkDerivation rec {
-  name = "rippled-validator-keys-tool-20180927-${builtins.substring 0 7 rev}";
-  rev = "d7774bcc1dc9439c586ea1c175fcd5ff3960b15f";
+  pname = "rippled-validator-keys-tool";
+  version = "0.3.2";
 
-  src = fetchgit {
-    url = "https://github.com/ripple/validator-keys-tool.git";
-    inherit rev;
-    sha256 = "1hcbwwa21n692qpbm0vqy5jvvnf4aias309610m4kwdsnzfw0902";
+  src = fetchFromGitHub {
+    owner = "ripple";
+    repo = "validator-keys-tool";
+    rev = "5d7efcfeda3bdf6f5dda78056004a7c326321e9b";
+    sha256 = "1irm8asp6plk9xw3ksf4fqnim8h0vj3h96w638lx71pga1h4zvmy";
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ openssl boost zlib rippled ];
+  buildInputs = [ openssl boost zlib icu rippled ];
 
   hardeningDisable = ["format"];
 
-  preConfigure = ''
-    export CXX="$(command -v $CXX)"
-    export CC="$(command -v $CC)"
-  '';
+  cmakeFlags = [
+    "-Dep_procs=1"
+  ];
 
   installPhase = ''
+    runHook preInstall
     install -D validator-keys $out/bin/validator-keys
+    runHook postInstall
   '';
 
   meta = with lib; {
```

Comment:
Maybe you could add yourself as a maintainer?

Reply:
I suppose I must (though I have no idea what this package does and have no desire to learn about cryptocurrencies)...

Reply:
You don't need to if you don't want to.

Reply:
No, it's fine. I'm just being dramatic.